app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,33 @@
+<% if page.main_properties_present? %>
+  <div class="crayons-card p-5">
+    <% if page.image %>
+      <div class="s:max-w-50 mb-4">
+        <img src="<%= page.image %>" height="auto" alt="Cover image">
+      </div>
+    <% end %>
+    <div>
+      <a href="<%= page.url %>" target="_blank" rel="noopener noreferrer">
```

Comment:
Based on the design (no underline, dark color text), I think we will need to add some styles here to override the default style for links inside of posts:

![title is underlined and branded color](https://user-images.githubusercontent.com/20773163/160883909-241ea3ea-7f35-4a40-a73e-b34fde33375d.png)

AFAIK I don't think we have a utility class for `text-decoration: none` (paging @ludwiczakpawel in case I am wrong) - if not, it probably belongs in `typography.scss`